src/ServiceControl.TransportAdapter/ServiceControl.TransportAdapter.csproj

```diff
@@ -38,13 +38,19 @@
   <PropertyGroup>
     <StartupObject />
   </PropertyGroup>
+  <PropertyGroup>
+    <SignAssembly>true</SignAssembly>
+  </PropertyGroup>
+  <PropertyGroup>
+    <AssemblyOriginatorKeyFile>NServiceBus.snk</AssemblyOriginatorKeyFile>
```

Comment:
I can't find the .snk file in the source. Is it there?
Also, I think the convention is keeping the NServiceBus.snk in the root of the src folder and reference that (don't think you can do it via VS UI and have to manually edit the .csproj)